inertia/utils.py

```diff
@@ -3,26 +3,42 @@
 from django.db.models.query import QuerySet
 from django.forms.models import model_to_dict as base_model_to_dict
 
+
 def model_to_dict(model):
-  return base_model_to_dict(model, exclude=('password',))
+    return base_model_to_dict(model, exclude=("password",))
+
 
 class InertiaJsonEncoder(DjangoJSONEncoder):
-  def default(self, value):
-    if isinstance(value, models.Model):
-      return model_to_dict(value)
-    
-    if isinstance(value, QuerySet):
-      return [model_to_dict(model) for model in value]
-    
-    return super().default(value)
+    def default(self, value):
+        if isinstance(value, models.Model):
+            return model_to_dict(value)
+
+        if isinstance(value, QuerySet):
+            return [model_to_dict(model) for model in value]
+
+        return super().default(value)
+
 
 class LazyProp:
-  def __init__(self, prop):
-    self.prop = prop
+    def __init__(self, prop):
+        self.prop = prop
+
+    def __call__(self):
+        return self.prop() if callable(self.prop) else self.prop
+
+
+class DeferredProp:
+    def __init__(self, prop, group=None):
+        self.prop = prop
+        self.group = group if group else ""
+
+    def __call__(self):
+        return self.prop() if callable(self.prop) else self.prop
 
-  def __call__(self):
-    return self.prop() if callable(self.prop) else self.prop
-  
 
 def lazy(prop):
-  return LazyProp(prop)
+    return LazyProp(prop)
+
+
+def defer(prop, group=None):
```

Comment:
The [laravel](https://github.com/inertiajs/inertia-laravel/blob/0259e37f802bc39c814c42ba92c04ada17921f70/src/ResponseFactory.php#L112) and [rails](https://github.com/inertiajs/inertia-rails/blob/8c749e32e568b4acb54357d05d9a85e2f12b6ea1/lib/inertia_rails/defer_prop.rb#L12) versions use a default group of "default", could we add that here?